src/sentry/api/endpoints/user_details.py

```diff
@@ -89,3 +97,74 @@ def put(self, request, user):
             return Response(serialize(user, request.user))
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @sudo_required
+    def delete(self, request, user):
+        """
+        Delete User Account
+
+        Also removes organizations if they are an owner
+        :pparam string user_id: user id
+        :param list organizations: List of organization ids to remove
+        :auth required:
+        """
+
+        # be strict since we're removing orgs
+        if 'organizations' not in request.DATA or not isinstance(
+                request.DATA.get('organizations'), list):
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+
+        # from `frontend/remove_account.py`
+        org_list = Organization.objects.filter(
+            member_set__role=roles.get_top_dog().id,
```

Comment:
i think we should just change this to be like member_set__role__in=[roles_with_org_delete_scope]